BACKEND/index.js

```diff
@@ -0,0 +1,65 @@
+// IMPORTING REQUIRED MODULES
+const mongoose = require('mongoose');
+const bodyParser = require("body-parser")
+const ejs = require("ejs");
+const express = require('express');
+const path = require('path');
+const app = express();
+
+
+
+const foods = require("./routers/food-routes.js");
+const users = require("./routers/user-routes.js");
+
+app.use("/canteen/food", foods);
+app.use("/canteen/user", users);
+
+
+
+
+// SELECTING THE PORT
+const port = 80;
+
+// IMPORTING SCHEMA OF FOOD ITEMS FROM "model.js"
+const Menu = require("./model.js");
+
+
+
+app.use(express.static("public"));
+
+
+
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+
+main().catch(err => console.log(err));
+
+
+// CONNECTING MONGOOSE AND MONGODB
+async function main() {
+  await mongoose.connect('mongodb://127.0.0.1:27017/canteen').then(() => {

```

Comment:
Change this to what it was before; @Aryan3902 got a bit confused, it was correct before.